tests/dtls_ciphers_util.c

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ *
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ *******************************************************************************/
+
+#include <string.h>
+
+#include "dtls_ciphers_util.h"
+
+
+struct cipher_entry {
+	const char* name;
+	const dtls_cipher_t cipher;
+};
+
+#define CIPHER_ENTRY(X) { .name = #X, .cipher = X }
+#define ARRAY_LENGTH (sizeof(map)/sizeof(struct cipher_entry))
+#define SEP ':'
+
+static const struct cipher_entry map[] = {
```

Comment:
Because of the same reason, as in many other cases.

it's easier to create something simple on your own than to understand a more or less complex solution of someone else :-).

I would prefer to leave this for now and after merging the pending PRs we may discus that topic again.